ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.ts

```diff
@@ -1,10 +1,14 @@
-import { Component, OnInit } from '@angular/core';
-import { RepresentativeBaseComponent } from '../representative-base-component/representative-base.component';
+import { Component } from '@angular/core';
+import { SuitabilityChoicePageBaseComponent } from '../../components/suitability-choice-page-base.component';
 
 @Component({
   selector: 'app-access-to-room',
   templateUrl: './access-to-room.component.html',
-  styles: []
+  styleUrls: ['./access-to-room.component.css']
 })
-export class AccessToRoomComponent extends RepresentativeBaseComponent {
+export class AccessToRoomComponent extends SuitabilityChoicePageBaseComponent {
+
```

Comment:
I know we're missing this on other pages as well but maybe it's better we start putting it in now to start with:
on ngInit can you set the choice value to the model room value?

Reply:
But it was set in bindModel() ? or do you mean something else?